manifest.yml

```diff
@@ -1,6 +1,9 @@
 ---
 default_config: &defaults
-  buildpack: https://github.com/cloudfoundry/multi-buildpack#v1.0.3
+  buildpacks:
+  # We need the "apt" buildpack to install a MySQL client for Drush
+    - https://github.com/cloudfoundry/apt-buildpack#v0.1.6
+    - https://github.com/cloudfoundry/php-buildpack#v4.3.72
```

Comment:
Is there a reason to pin the buildpack versions here? The obsolete multi-buildpacks buildpack needed that, but it's not necessary in the native `buildpacks:` directive that succeeded it. See https://github.com/18F/cf-ex-drupal8/pull/26/files ; I will rebase it on this branch.

Reply:
I am just using what people have set here before, and not knowing if they were important, I decided to get it going with whatever requirements the other folks had set in the past.  We can certainly do that in a later PR.

Reply:
Handled in https://github.com/18F/cf-ex-drupal8/pull/26